packages/data/src/options/resolvers.js

```diff
@@ -23,3 +23,23 @@ export function* getOptionsWithRequest( names ) {
 		return error;
 	}
 }
+
+const fetches = {};
+
+export function* getOption( name ) {
+	yield setIsRequesting( name );
+	const names = yield getOptionsToRequest( name );
+
+	const fetchInProgress = fetches[ names ];
+	if ( fetchInProgress ) {
+		return;
```

Comment:
Just a heads up that I added this back in since we need to return early here.

Returning `yield apiFetch( { path: url } );` will hit the options endpoint again for each request.

Reply:
As Paul pointed out, this will cause the built in cached resolver for `getOption( name )` to be marked as finished - so this effectively means you cannot depend on the wp.data resolution state for any of the options to be accurate. This could be a gotcha for folks not realizing `getOption` has a special resolution state.